codespell_lib/data/dictionary_capitalization.txt

```diff
@@ -0,0 +1,4 @@
+aws->!AWS
+github->!GitHub
+ip->!IP
+tcp->!TCP
```

Comment:
For some others you could search the dictionary for where the RHS is in all caps, e.g. CMYK.